cg/cli/demultiplex/finish.py

```diff
@@ -45,13 +49,27 @@ def post_process_illumina_run(context: CGConfig, demultiplexed_run_dir_name: str
 @DRY_RUN
 def post_process_all_illumina_runs(context: CGConfig, dry_run: bool):
     """Command to post-process all demultiplexed Illumina runs."""
-
-    post_processing_service = IlluminaPostProcessingService(
-        status_db=context.status_db,
-        housekeeper_api=context.housekeeper_api,
-        dry_run=dry_run,
-        demultiplexed_runs_dir=Path(context.run_instruments.illumina.demultiplexed_runs_dir),
+    directories: list[Path] = get_directories_in_path(
+        Path(context.run_instruments.illumina.demultiplexed_runs_dir)
     )
-    is_error_raised: bool = post_processing_service.post_process_all_runs()
+    is_error_raised: bool = False
+    for directory in directories:
```

Comment:
I don't think this logic should be here. If what is different is the way we get the datetimes, we could implement the distinction way down and let the post-processing be agnostic to the flow cell type